google-apps-script.js

```diff
@@ -21,14 +25,23 @@ function doPost(e) {
   try {
     Logger.log(e); // the Google Script version of console.log see: Class Logger
     record_data(e);
-
-    var mailData = e.parameters; // just create a slightly nicer variable name for the data
+    
+    // shorter name for form data
+    var mailData = e.parameters;
+    
+    // names and order of form elements
+    var dataOrder = JSON.parse(e.parameters.formDataNameOrder);
+    
+    // determine recepient of the email
+    // if you have your email uncommented above, it uses that `TO_ADDRESS`
+    // otherwise, it defaults to the email provided by the form's data attribute
+    var sendEmailTo = (typeof TO_ADDRESS !== "undefined") ? TO_ADDRESS : mailData.formGoogleSendEmail;
```

Comment:
@mckennapsean we just need to _warn_ people that if they put a form on a website that allows people to send email to "_arbitrary_" email address it will _eventually_ be abused.
The work-around I was considering was always "CC-ing" the owner of the script so they can keep an eye on it. 😉